ClassicPress-research/classic-commerce
# Change WC to CC on Rest-API page in advanced settings

### All Submissions:


### Changes proposed in this Pull Request:

Change one occurrence of WC to CC on the rest api page in settings, in the file: class-wc-admin-api-keys.php

### How to test the changes in this Pull Request:

1. Check out PR change to class-wc-admin-api-keys.php
2. Copy this file to a test site and go to settings -> advanced -> REST API screen

### Other information:


### Changelog entry

Change WC to CC on Rest-API page in advanced settings.
